sdk/selfhelp/Azure.ResourceManager.SelfHelp/tests/TroubleshooterTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.ResourceManager.SelfHelp.Tests
+{
+    using Azure.Core.TestFramework;
+    using Azure.Core;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+    using System;
+    using Azure.ResourceManager.SelfHelp.Models;
+    using System.Collections.Generic;
+
+    public class TroubleshooterTests : SelfHelpManagementTestBase
+    {
+        private bool isAsync = true;
+        public TroubleshooterTests(bool isAsync) : base(isAsync)//, RecordedTestMode.Record)
```

Comment:
@archerzz I didn't follow, RecordedTestMode.Record is commented out. So by default it should take Playback mode right?

Reply:
> @archerzz I didn't follow, RecordedTestMode.Record is commented out. So by default it should take Playback mode right?

@sanar-microsoft Maybe I was looking at an old commit. Anyway, it's fine now. Thanks.